pulsar-client/src/main/java/org/apache/pulsar/client/api/Schema.java

```diff
@@ -18,10 +18,14 @@
  */
 package org.apache.pulsar.client.api;
 
+import org.apache.pulsar.common.schema.SchemaInfo;
+
 public interface Schema<T> {
     byte[] encode(T message);
     T decode(byte[] bytes);
 
+    SchemaInfo getSchemaInfo();
```

Comment:
If `SchemaInfo` becomes part of API, we should move it in the same `..client.api` package

Reply:
It's used in multiple places. Are you suggesting I keep it in common but add it to the client package?